### DISCUSSION TITLE
Multi Vector Search Functionality

### DISCUSSION THREAD

**Discussion by dee-corr** [milvus-io/milvus#20944] [Q&A and General discussion] [answered]
I'm wondering if there is any update on providing the ability to search records based on two vector fields simultaneously?

Thanks

**Comment by yhmo** [comment, marked as answer]
Each search request is applied to a single collection, and Milvus doesn't support multi-vector-field in one collection. So, currently, Milvus has no ability to search on two vector fields.
In future, we may support multi-vector-field in one collection, but it is a long-term feature.

**Comment by dee-corr** [comment]
Thanks @yhmo
